src/EnlistmentInfo.targets

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Thanks. I'll try to find time to learn more about it and try it out. But IMO it would have to be _wicked awesome_ to overrule my personal devotion to "use what's already on every (or 99%) of developers machines."